website/description_issues.qmd

```diff
@@ -25,3 +25,18 @@ Description: The goal of 'readr' is to provide a fast and friendly way to
     to flexibly parse many types of data found in the wild, while still
     cleanly failing when data unexpectedly changes.
 ```
+
+
+# Explaining Acronyms
+
+## Problem
+
+You are using acronyms in your DESCRIPTION file or they are flagged by the automatic spell check..
+
+## Solution
+
+Document all non-obvious acronyms in the [cran-comments.md](https://usethis.r-lib.org/reference/use_cran_comments.html) file to facilitate the CRAN team review.
```

Comment:
This assumes using devtools/usethis to submit to CRAN. Is this intended? If so it might be best to document it (here or in general in the book).

Reply:
It was mentioned but not in general, only in a Note. If one has only this issue and comes here to read it it won't know about devtools submission process, which can lead to more confusion. At least I would provide a link to it or a footnote.